llvm/include/llvm/ADT/FunctionExtras.h

```diff
@@ -317,8 +317,10 @@ template <typename ReturnT, typename... ParamTs> class UniqueFunctionBase {
     // Clear the old callback and inline flag to get back to as-if-null.
     RHS.CallbackAndInlineFlag = {};
 
-#ifndef NDEBUG
-    // In debug builds, we also scribble across the rest of the storage.
+#if !defined(NDEBUG) && !defined(LLVM_ADDRESS_SANITIZER_BUILD)
```

Comment:
it's always defined, should be 
```
#if !defined(NDEBUG) && !LLVM_ADDRESS_SANITIZER_BUILD
```

Reply:
and #include "llvm/Support/Compiler.h" to be sure

Reply:
Thx, done!